game-frontend/src/components/canvas-objects/Unit.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { FC } from 'react';
```

Comment:
I would not change the unit visualization.
I think adding icons for the powerups should be enough.